nginx/config.go

```diff
@@ -205,6 +205,11 @@ http {
 			allow 127.0.0.1;
 			deny all;
 		}
+    location /nginx_status {
+      stub_status on;
+      allow 127.0.0.1;
+      deny all;
+    }
```

Comment:
@pixeleet if you can just make the leading whitespace consistent with what's above and below, then I am LGTM on this.